data/Localization/german_(germany)/global.ini

```diff
@@ -39707,79 +39707,79 @@ ServiceBeacon_Revive_Obj_Title=Rescue ~serviceBeacon(InitiatorName)
 ShipName_Test_001=Freiberufler
 ShipName_Test_002=Starfarer
 ShipName_Test_003=Konstellation
-ShipSelector_ButtonTitle_Deliver=Lieferung anfordern
-ShipSelector_ButtonTitle_FileClaim=Einen Anspruch anmelden
-ShipSelector_ButtonTitle_PayDeductible=Selbstbehalt zahlen
-ShipSelector_ButtonTitle_PayExpedited=Expedited-Gebühr bezahlen
+ShipSelector_ButtonTitle_Deliver=Bestellen
```

Comment:
How about ShipSelector_ButtonTitle_Deliver=Anfordern

Reply:
Yes, it fits and was adopted.